pkg/sync/rpc/server.go

```diff
@@ -146,9 +150,7 @@ func NewSyncAgentServer(startPort, endPort int, replicaAddress string) *SyncAgen
 		BackupList: &BackupList{
 			RWMutex: sync.RWMutex{},
```

Comment:
As I mentioned before, those RWMutex initialization are not necessary as well.

Reply:
Sorry, my bad. There is one place using the NewPurgeStatus:
https://github.com/longhorn/longhorn-engine/blob/4bf6f2b65fc8b64898ff76b788fe1dfb30d54003/pkg/sync/rpc/server.go#L833

Reply:
This can be directly assigned too since we no longer do special initialization for PurgeStatus.

Reply:
Hmm? Where is this part?

Reply:
Can you add another commit to clean up those unnecessary RWMutex initializations?